cvat/apps/dataset_manager/bindings.py

```diff
@@ -1907,25 +2000,29 @@ def _clean_display_message(self) -> str:
             message = "Dataset must contain a file:" + message
         return re.sub(r' +', " ", message)
 
-def mangle_image_name(name: str, subset: str, names: defaultdict[tuple[str, str], int]) -> str:
-    name, ext = name.rsplit(osp.extsep, maxsplit=1)
+def mangle_image_name(name: str, subset: str, names: defaultdict[tuple[str, str], int], task_id: Optional[int] = None) -> str:
```

Comment:
```suggestion
def mangle_image_name(name: str, subset: str, names: defaultdict[tuple[str, str], int], task_id: int | None = None) -> str:
```

We're not using this old syntax anymore.